src/rpm/package.rs

```diff
@@ -110,8 +110,7 @@ impl RPMPackage {
         let rsa_signature_spanning_header_and_archive =
             signer.sign(&mut header_and_content_cursor)?;
 
-        // TODO FIXME verify this is the size we want, I don't think it is
```

Comment:
This was verified I assume?

Reply:
I cross checked, this is indeed correct, I have a changeset ready once this an #20 are merged

Reply:
The code logic here is correct,remove the TODO FIXME tag here to avoid confusing, see: https://refspecs.linuxbase.org/LSB_3.1.1/LSB-Core-generic/LSB-Core-generic/pkgformat.html